pysat/instruments/methods/nasa_cdaweb.py

```diff
@@ -70,29 +72,26 @@ def list_files(tag=None, sat_id=None, data_path=None, format_str=None,
         list_files = functools.partial(cdw.list_files,
                                        supported_tags=supported_tags)
 
+    Note
+    ----
+    This function will move to pysat.instruments.methods.general.list_files in
```

Comment:
Move sounds to me like a copy of the method isn't already over in the new location. Perhaps something like, 
'This method has been moved to..... The copy here will be removed in pysat 3.0.

Reply:
Looking at how we've handled this elsewhere, what about:

methods.nasa_cdaweb.list_files has been deprecated and will be removed in pysat 3.0.0.  Please use methods.general.list_files instead.